ispcrt/detail/Device.h

```diff
@@ -33,6 +33,8 @@ struct Device : public RefCounted {
     virtual void *deviceNativeHandle() const = 0;
     virtual void *contextNativeHandle() const = 0;
 
+    virtual ISPCRTDeviceType _getDeviceType() const = 0;
```

Comment:
Why not just simply "getType()" since this method of Device?

Reply:
I agree. This would be better much better than leading underscore. My second preference would be `Impl` suffix, but just `getType()` looks much better.